packages/devextreme/js/__internal/grids/new/card_view/widget.test.ts

```diff
@@ -65,3 +65,43 @@ describe('regressions', () => {
     expect(cardView.option('pager.showPageSizeSelector')).toBe(false);
   });
 });
+
+describe('editing validation', () => {
+  const checkError = (error: { __id: string; __details: string }): void => {
+    expect(error.__id).toBe('E1042');
+  };
+
+  it.only('should throw E1042 error when no keyExpr and clicking on add/edit/delete', async () => {
```

Comment:
Now only this test running on CI

```suggestion
  it('should throw E1042 error when no keyExpr and clicking on add/edit/delete', async () => {
```

Reply:
done ✔️